shared/haml/hamburger.haml

```diff
@@ -92,17 +110,24 @@
 
 #hamburger{class: hamburger_class}
   %ul.hide-responsive-menu
-    .user-links
+    .user-links{class: show_user_options}
       - if user_type == "teacher"
         - teacher_entries.each do |entry|
           %li{class: show_teacher_options}
             %a{href: entry[:url]}= entry[:title]
+        %div{class: "divider"}
       - elsif user_type == "student"
         - student_entries.each do |entry|
           %li{class: show_student_options}
             %a{href: entry[:url]}= entry[:title]
+        %div{class: "divider"}
+      - else
+        - signed_out_entries.each do |entry|
+          %li{class: show_signed_out_options}
+            %a{href: entry[:url]}= entry[:title]
+        %div{class: "divider"}
```

Comment:
An alternative would be to check for type being any of teacher/student/signed_out and then show the divider.